backend/de.metas.payment.revolut/src/main/java/de/metas/payment/revolut/model/RevolutExportCSVColumn.java

```diff
@@ -34,7 +34,7 @@
 	AccountNo("Account number"),
 	RoutingNo("Sort code or Routing number"),
 	IBAN("IBAN"),
-	BIC("BIC"),
```

Comment:
AFAIU
- BIC is the name of the column in the revolut standard.
- BIC is practically the same as switftcode

=> therefore, please leave the name as BIC, at least in the csv output file